Parser/action_helpers.c

```diff
@@ -1495,7 +1495,8 @@ expr_ty _PyPegen_formatted_value(Parser *p, expr_ty expression, Token *debug, ex
         if (PyUnicode_GET_LENGTH(conversion->v.Name.id) > 1 ||
             !(first == 's' || first == 'r' || first == 'a')) {
             RAISE_SYNTAX_ERROR_KNOWN_LOCATION(conversion,
-                                              "f-string: invalid conversion character: expected 's', 'r', or 'a'");
+                                              "f-string: invalid conversion character %R: expected 's', 'r', or 'a'",
```

Comment:
As we are removing `f-string:` from the other tests should we remove it from here as well?

Reply:
I guess there's also the question, should we maybe find a way to keep `f-string` in all the error messages that come from inside an f-string?

Reply:
My reasoning was, errors that can happen in a normal code should not have f-string: in front of them (e.g. `1.X()`  vs `f"{1.X()}"`) but errors that are specific to f-strings (like this one or the error we give when saying unmatched brace even though there was a previous brace f"{)}" vs "{)}") should include `f-string: ` prefix to denote that this rule only applies within the f-string context. But am open to other ideas as well (fully removing or automatically adding `f-string: ` at the top level error delegating part).

Reply:
I think I prefer to remove all the `f-string:` based on two things:

* We don't mention other constructs elsewhere (as in "error in import statement") or something like that.
* The parser doesn't really know that some syntax error is really inside an f-string. Sure, it can know what is happening inside one but it doesn't need to know how deep into it is. 
* I don't really think it helps a lot the user once is clear where the error is when is printed.

Reply:
Ok, let's go with no `f-string: ` then. While I agree with @isidentical's view and it makes sense, I think in this case consistency is a bit more important, so let's remove them from everywhere if you're all alright with it.

Reply:
I am happy to to go the other way though if you all agree that is helpful 😅 

@isidentical @mgmacias95 thoughts?

Reply:
Removing `f-string: ` is the much simpler option, and I think I'd be down with it considering the prefix doesn't add that much information and we can always provide better messages when needed. If everyone agrees, let me know and I'll push a change that removes all the prefixes in the code and the tests

Reply:
Ok, let's make a quick poll. React to this comment with 👍 for removing the prefix and 👎 to keep it only on f-string related errors.

Reply:
agreed, add some context saying it's a f-string in these cases can be very helpful. I also agree on adding the context in the error message and not as a `f-string:` prefix.

Reply:
Ok, this is what we are going to do: let's move this forward and let's make another PR to "normalize" the error messages.